contrib/test/integration/build/kubernetes.yml

```diff
@@ -43,9 +43,9 @@
       export CONTAINER_RUNTIME_ENDPOINT='{{ crio_socket }} --runtime-request-timeout=5m'
       export ALLOW_SECURITY_CONTEXT=","
       export ALLOW_PRIVILEGED=1
-      export DNS_SERVER_IP={{ ansible_eth0.ipv4.address }}
-      export API_HOST={{ ansible_eth0.ipv4.address }}
-      export API_HOST_IP={{ ansible_eth0.ipv4.address }}
+      export DNS_SERVER_IP={{ ansible_default_ipv4.address }}
+      export API_HOST={{ ansible_default_ipv4.address }}
+      export API_HOST_IP={{ ansible_default_ipv4.address }}
```

Comment:
any reason to modify these?

Reply:
I happened to be trying to run the playbook against a system where the default network interface wasn't named "eth0".  This fixed it for me.

Reply:
makes sense, I think there's no harm in having this then